src/Parsers/Lexer.cpp

```diff
@@ -385,10 +378,10 @@ Token Lexer::nextTokenImpl()
                     return Token(TokenType::DollarSign, token_begin, ++pos);
                 }
             }
-            if (isWordCharASCII(*pos) || *pos == '$')
+            if (isWordCharASCII(*pos))
```

Comment:
Although `$` is already gets widely used, let's keep it.

Reply:
done